### DISCUSSION TITLE
Option "Enable VIN Not Send Data"

### DISCUSSION THREAD

**Discussion by futuretristan** [tautcony/ISTA-Patcher#12] [Q&A] [answered]
Hi, can someone explain to me what option "Enable VIN Not Send Data" is actually doing and when it should be set to "true"?

https://github.com/tautcony/ISTA-Patcher/blob/80c142f1b615cb6f49401c8e657660f94a422216/ISTA-Patcher/ProgramArgs.cs#L31C15-L31C15

**Comment by tautcony** [comment, marked as answer]
This feature itself a GDRP related function to control whether or not to transmit FASTA data and OBFCM data, but for the case of using this program, it is not connected to the server at all! Actually, it's just writing for fun.
